src/compiler/transformers/component-native/proxy-custom-element-function.ts

```diff
@@ -47,8 +47,22 @@ export const proxyCustomElement = (
               continue;
             }
 
+            // to narrow the type of `declaration.initializer` to `ts.ClassExpression`
+            if (!ts.isClassExpression(declaration.initializer)) {
+              continue;
+            }
```

Comment:
This feels a little sketchy to me. Is there a different way to do this type-cast/assertion?

Reply:
We could instead do `if (ts.isClassExpresion(declaration.initializer)) {` and then put all the code within the `if` block - that essentially amounts to the same thing though, just changing a lot more lines. I don't think typically we would get to this point and have `declaration.initializer` not be the thing we want but I thought there's no harm in a little more type safety here. What's your concern exactly?

Reply:
Not necessarily concerned, this just stood out to me as a pattern I haven't really seen. I've always strayed away from using `continue`. I've not too worried if you and @rwaskiewicz  are cool with it

Reply:
I'm OK with this as-is. Using an if statement with a `continue` achieves the same thing as `if (ts.isClassExpresion(declaration.initializer)) {`, just with less nesting. I don't think there's a better (safer) way to narrow the type here. I (personally) prefer less nesting, if only for less lines of code changed.